dist/s_string.ok

```diff
@@ -499,8 +497,6 @@ asm
 aspell
 assertfmt
 async
```

Comment:
Curious that there is a use of `async` in the source tree that's not related to async code?

Reply:
It is needed for the `NEWS` file that contains content about what is in each of the various releases, including the old ones where this was added. And that file goes through the string check. It surprised me too.